ballerina/types.bal

```diff
@@ -259,3 +259,560 @@ public enum LogLevel {
+    @display {label:"Support Ordering"}
+    boolean supportOrdering?;
+};
+# Upadate Topic Propertise
+#
+# + autoDeleteOnIdle - ISO 8601 timeSpan idle interval after which the queue is automatically deleted. The minimum duration is 5 minutes.
+# + defaultMessageTimeToLive - ISO 8601 default message timespan to live value. This is the duration after which the message expires, starting from when the message is sent to Service Bus.
+# + duplicateDetectionHistoryTimeWindow - ISO 8601 timeSpan structure that defines the duration of the duplicate detection history. The default value is 10 minutes.
+# + maxDeliveryCount - The maximum delivery count. A message is automatically deadlettered after this number of deliveries. Default value is 10.
+# + maxMessageSizeInKilobytes - The maximum size of the queue in megabytes, which is the size of memory allocated for the queue.
+# + maxSizeInMegabytes - The maximum size of the queue in megabytes, which is the size of memory allocated for the queue.
+# + status - Enumerates the possible values for the status of a messaging entity.
+# + userMetadata - Metadata associated with the queue.
+# + enableBatchedOperations - Value that indicates whether server-side batched operations are enabled.
+# + deadLetteringOnMessageExpiration - Value that indicates whether this queue has dead letter support when a message expires.
+# + requiresDuplicateDetection - Value indicating if this queue requires duplicate detection.
+# + enablePartitioning - Value that indicates whether the queue is to be partitioned across multiple message brokers.
+# + requiresSession - Value that indicates whether the queue supports the concept of sessions.
+# + supportOrdering - Defines whether ordering needs to be maintained.
+public type UpdateTopicOptions record {
+    @display {label:"Auto Delete On Idle"}
+    Duration autoDeleteOnIdle?;
+    @display {label:"Default Message Time To Live"}
+    Duration defaultMessageTimeToLive?;
+    @display {label:"Duplicate Detection History Time Window"}
+    Duration duplicateDetectionHistoryTimeWindow?;
+    @display {label:"Max Delivery Count"}
+    int maxDeliveryCount?;
+    @display {label:"Max Message Size In Kilobytes"}
+    int maxMessageSizeInKilobytes?;
+    @display {label:"Max Size In Megabytes"}
+    int maxSizeInMegabytes?;
+    @display {label:"Status"}
+    EntityStatus status?;
+    @display {label:"User Metadata"}
+    string userMetadata?;
+    @display {label:"Enable Batched Operations"}
+    boolean enableBatchedOperations?;
+    @display {label:"Dead Lettering On Message Expiration"}
+    boolean deadLetteringOnMessageExpiration?;
+    @display {label:"Requires Duplicate Detection"}
+    boolean requiresDuplicateDetection?;
+    @display {label:"Enable Partitioning"}
+    boolean enablePartitioning?;
+    @display {label:"Requires Session"}
+    boolean requiresSession?;
+    @display {label:"Support Ordering"}
+    boolean supportOrdering?;
+};
+# Topic List
```

Comment:
Missing blank line between record definitions. Fix all the similar places

Reply:
Addressed in [f0bbe24](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/180/commits/f0bbe24dab6c69c66ba7ae68ca343a18c0eaf18b).